Documentation/subcommands/stop.md

```diff
@@ -0,0 +1,11 @@
+# rkt stop
```

Comment:
Do we actually document somewhere the other way of stopping a rkt pod?

- `^]^]^]` for stage1 with nspawn.
- `^A-x` for stage1 with kvm.
- `machinectl poweroff`, `machinectl terminate` if using systemd with machine registration.
- `systemctl stop` if the rkt pod has been started as a systemd service.